prow/cmd/peribolos/README.md

```diff
@@ -31,6 +31,8 @@ orgs:
     - bob
     admins:
     - carl
+    outside_collaborators:
```

Comment:
I do not think this is the right pattern. As you mention later, this does not result in adding outside_collaborators.

Instead we should add a `Repos map[string]Repo` field to the `org.Config` struct https://github.com/kubernetes/test-infra/blob/1b1b232df708eb50aca08009757d4d91c66f3a7c/prow/config/org/org.go#L40-L45

This has multiple benefits:
* Allows us to control who has what rights to repos in the org
* Allows us to manage repo collaborators

We can then walk this config adding all people into a set, and subtract this set from the set of outside collaborators. Anyone who is not present in this config can be removed (although I suspect it might happen automatically when we delete them as repo collaborators).